math/mathcore/test/testVectorizedTMath.cxx

```diff
@@ -0,0 +1,108 @@
+#include <cmath>
+#include <random>
+#include "VectorizedTMath.h"
+#include <gtest/gtest.h>
+
+#if defined(R__HAS_VECCORE) && defined(R__HAS_VC)
+
+Double_t uniform_random(Double_t a, Double_t b)
+{
+   return a + (b - a) * drand48();
+}
+
+class VectorizedTMathTest : public ::testing::Test {
+protected:
+   VectorizedTMathTest() {}
+
+   size_t kVS = vecCore::VectorSize<ROOT::Double_v>();
+   Double_t input_array1[16384] __attribute__((aligned(VECCORE_SIMD_ALIGN)));
+   Double_t input_array2[16384] __attribute__((aligned(VECCORE_SIMD_ALIGN)));
+
+   Double_t output_array[16384] __attribute__((aligned(VECCORE_SIMD_ALIGN)));
```

Comment:
Can you make this a `#define N 16384` or `static const int N = 16384;` at the beginning of the file? Then we can change in one place if needed.